kotlin-inject-compiler/test/src/test/kotlin/me/tatarka/inject/test/RoundsTest.kt

```diff
@@ -41,13 +40,13 @@ class RoundsTest {
                     }
                 """.trimIndent()
             ).compile()
-        }.isSuccess()
+        )
```

Comment:
since we aren't actually asserting anything `assertThat` isn't doing anything here. Maybe add a comment that we expect it not to throw instead

Reply:
How about `assertDoesNotThrow`?

Reply:
I mean if it throws it'll fail the test anyway, no strong opinions though

Reply:
OK pushed update with it. I think `assertDoesNotThrow` provides a better UX vs just throwing.